tests/hexahedron_element_test.cc

```diff
@@ -764,6 +764,18 @@ TEST_CASE("Hexahedron elements are checked", "[hex][element][3D]") {
       REQUIRE(indices(11, 1) == 1);
       REQUIRE(indices(11, 2) == 2);
     }
+
+    SECTION("Eight noded hexahedron shape function for face indices") {
+      // Check for face indices
+      unsigned face_id = 0;
+      Eigen::VectorXi indices = hex->face_indices(face_id);
+      REQUIRE(indices.size() == 4);
+
+      REQUIRE(indices(0) == 0);
```

Comment:
Don't you have to check all of the faces?

Reply:
OK! I will check for all faces then